components/merkledb/benches/lib.rs

```diff
@@ -14,15 +14,12 @@
 
 use criterion::{criterion_group, criterion_main};
 
-use crate::{
+use crate::benchmarks::{
     encoding::bench_encoding, schema_patterns::bench_schema_patterns, storage::bench_storage,
     transactions::bench_transactions,
 };
 
-mod encoding;
```

Comment:
I know it sounds odd, but I couldn't use `BenchDB` declared in `benches/lib.rs` in the modules. Possibly, it's because of some specific of files structure in benches.